DEV-GUIDE.md

```diff
@@ -1,55 +1,37 @@
- 
+
 <p>
-  <img src="https://github.com/inspetor/slate/blob/master/source/images/logo-color.png" width="200" height="40" alt="Inspetor Logo"> </img> 
+  <img src="https://github.com/inspetor/slate/blob/master/source/images/logo-color.png" width="200" height="40" alt="Inspetor Logo"> </img>
 </p>
 
 # Inspetor Antifraud
-Antrifraud Inspetor library for PHP. 
-
-## Description
-This READ ME file is special! It should help you, my dear developer, on your "*publishing library road*". Let's chat about library development and publishing new versions 'cause the war against fraud never ends.
+Developer README for the Inspetor PHP SDK. Contains dev setup information and publication instructions.
 
-## Setup Guide
-This is a step-by-step Inspetor publication guide:
 
 ### Development
-Ok, so you decided to make changes in this beautiful library. No worries, I appreciate that.
-Well, you should clone this repo:
+Requirements:
+- Composer
+- PHP > 7.1.3
+
+First step:
 ```
 git clone https://github.com/inspetor/inspetor-php.git
 ```
-And when it's done, you can start to code. Every useful information to the development of this library can be found [here](https://github.com/inspetor/inspetor-php/blob/master/README.md), general information about Inspetor [here](https://inspetor.github.io/slate/) and [here](https://github.io/inspetor/libraries) are some libraries definitions (swagger).
+Install dependencies:
+```
+composer install
+```
+Now rock 'n' roll!
 
 ### TEST YOUR STUFF!
 After finishing your code, it's simple to test:
-Locally: 
+Locally:
 ```
 ./vendor/bin/phpunit
 ```
-Integration tests: 
+Integration tests:
 ```
-./vendor/bin/phpunit --configuration integration-test.xml 
+./vendor/bin/phpunit --configuration integration-test.xml
```

Comment:
Nit: What you think of putting a line remebering people to check the staging `atomic.actions`?